server/controllers/authentication.coffee

```diff
@@ -9,9 +9,15 @@ localization = require '../lib/localization_manager'
 passwordKeys = require '../lib/password_keys'
 
 
-module.exports.registerIndex = (req, res) ->
+module.exports.registerIndex = (req, res, next) ->
     User.first (err, user) ->
-        if user?
+        if err?
+            error = new Error err.code
+            error.status = 500
+            error.template =
+                name: 'error'
```

Comment:
For user, proxy displays : 
![error_proxy](https://cloud.githubusercontent.com/assets/2659119/9766375/d30f6e4e-5718-11e5-9673-a96672f1d7a8.png)
Do you want I create a specific error page when proxy can't retrieve user ?